.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,24 @@
-[comment]: # (Please replace ... with your information. Remove < and >)
 [comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
-
-If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number
```

Comment:
We don't have bounties now

Reply:
We actually have means of compensating work from external contribution. This has been done during 2024 and there are at least two developers that I know interested in that.

But I think we should keep the removal as you did. This is unnecessary noise.

Reply:
@ilmotta is there a public way to compensate external contributors? I know a few that'd be interested on it

Reply:
Let's talk in private about this @ulisesmac, there are some important details first. If anybody is interested to understand this point, please reach out to me.